src/components/common/TransactionalFolioNumber.vue

```diff
@@ -54,7 +54,7 @@ export default class TransactionalFolioNumber extends Mixins(CommonMixin) {
 
   /** True if this component is valid. */
   get isValid (): boolean {
-    return (!this.doValidate || this.$refs.folioNumberInput?.valid)
+    return (!this.doValidate || !!this.$refs.folioNumberInput?.valid)
```

Comment:
This is just to change the response to a boolean (instead of, sometimes, undefined, which is falsy anyway... but testing).